test/db.tests.js

```diff
@@ -612,6 +701,33 @@ describe('LimitDBRedis', () => {
         });
       });
     });
+
+    it('should use refillRateMsPerToken config override when provided', (done) => {
```

Comment:
Didn't make sense to test bucket size on `wait` operations since 1 is the default.  Rate seemed to be the only configuration override of importance here.